docs/guides/how_to_guides/creating_batches/how_to_load_a_database_table_view_or_query_result_as_a_batch.rst

```diff
@@ -5,134 +5,246 @@ How to load a database table, view, or query result as a batch
+            If you have not already created a suite, you can do so now.
 
-        # Note, you can add the "overwrite_existing" flag to the below command if the suite
-        # exists but you would like to replace it.
-        context.create_expectation_suite(expectation_suite_name)
+            .. code-block:: python
 
+                # Note, you can add the "overwrite_existing" flag to the below command if the suite
+                # exists but you would like to replace it.
+                context.create_expectation_suite(expectation_suite_name)
 
-#. **Get the batch to validate**.
 
-    .. code-block:: python
+        #. **Get the batch to validate**.
 
-        batch = context.get_batch(
-            batch_kwargs=batch_kwargs,
-            expectation_suite_name=expectation_suite_name
-        )
+            .. code-block:: python
 
+                batch = context.get_batch(
+                    batch_kwargs=batch_kwargs,
+                    expectation_suite_name=expectation_suite_name
+                )
 
-Now that you have a Batch, you can use it to create Expectations or validate the data.
 
+        Now that you have a Batch, you can use it to create Expectations or validate the data.
 
-Additional Notes
-----------------
-  * If you are using Snowflake, and you have lowercase table or column names:
-    * If you are loading your batch with a table, you can use pass `"use_quoted_name":True` into your `batch_kwargs` dictionary. This will use the SQL Alchemy quoted_name method to ensure case sensitivity for your table and column names.
-    * If you are loading your batch with a query, if you have lowercase column names, you still need to pass `"use_quoted_name":True` into your `batch_kwargs` dictionary. You will also need to wrap your query in single quotes, and your table or column name in double quotes like so:
-        .. code-block:: python
 
-            batch_kwargs = {
-                ...
-                "use_quoted_name": True,
-                "query: 'select "lowercase_column_one", "lowercase_column_two" from "lowercase_table_name" limit 100'
-                ...
-            }
-  * For more information on configuring a Batch Kwargs generator, please see the relevant guides. The above code snippets use the following configuration:
+        Additional Notes
+        ----------------
+            * If you are using Snowflake, and you have lowercase table or column names:
+            * If you are loading your batch with a table, you can use pass `"use_quoted_name":True` into your `batch_kwargs` dictionary. This will use the SQL Alchemy quoted_name method to ensure case sensitivity for your table and column names.
+            * If you are loading your batch with a query, if you have lowercase column names, you still need to pass `"use_quoted_name":True` into your `batch_kwargs` dictionary. You will also need to wrap your query in single quotes, and your table or column name in double quotes like so:
```

Comment:
```suggestion
            * If you are using Snowflake, and you have lowercase table or column names:
              * If you are loading your batch with a table, you can use pass `"use_quoted_name":True` into your `batch_kwargs` dictionary. This will use the SQL Alchemy quoted_name method to ensure case sensitivity for your table and column names.
              * If you are loading your batch with a query, if you have lowercase column names, you still need to pass `"use_quoted_name":True` into your `batch_kwargs` dictionary. You will also need to wrap your query in single quotes, and your table or column name in double quotes like so:
```